src/fillTank.test.js

```diff
@@ -1,11 +1,84 @@
-'use strict';
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+describe('fillTank', () => {
+  const { fillTank } = require('./fillTank');
+  const customer = {
+    money: 3000,
+    vehicle: {
+      maxTankCapacity: 40,
+      fuelRemains: 8,
+    },
+  };
+  const fuelPrice = 4.80;
 
+  it('should be declared', () => {
+    expect(fillTank).toBeInstanceOf(Function);
   });
 
   // write tests here
+  it('should round the poured amount to the tenth part', () => {
+    fillTank(customer, fuelPrice, 20.78);
+
+    expect(customer.vehicle.fuelRemains).toStrictEqual(28.7);
+  });
+
+  it('should order a full tank if the amount is not given', () => {
+    fillTank(customer, fuelPrice);
```

Comment:
The test 'should order a full tank if the amount is not given' does not check if the customer's money was debited correctly. It's important to verify all side-effects a function may have.